src/test/java/com/notably/model/block/BlockTreeTest.java

```diff
@@ -0,0 +1,104 @@
+package com.notably.model.block;
+
+import static com.notably.testutil.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.notably.commons.core.path.AbsolutePath;
+import com.notably.commons.core.path.exceptions.InvalidPathException;
+import com.notably.model.block.exceptions.CannotModifyRootException;
+import com.notably.model.block.exceptions.DuplicateBlockException;
+import com.notably.model.block.exceptions.NoSuchBlockException;
+
+public class BlockTreeTest {
+    private BlockTree blockTree = new BlockTreeImpl();
+    private AbsolutePath toRoot;
+    private AbsolutePath toCs2103;
+    private AbsolutePath toCs2103Week1;
+    private AbsolutePath toCs2103Week1Lecture;
+
+    @BeforeEach
```

Comment:
I believe you should use the `@BeforeAll` annotation instead since you don't really need to recalculate these before each test.